images/win/scripts/Tests/Android.Tests.ps1

```diff
@@ -0,0 +1,26 @@
+Import-Module (Join-Path $PSScriptRoot "..\SoftwareReport\SoftwareReport.Android.psm1") -DisableNameChecking
+
+Describe "Android SDK" {
+    $androidToolset = (Get-ToolsetContent).android
+    $androidInstalledPackages = Get-AndroidInstalledPackages
+
+    $platformTestCases = @()
+    $platformList = $androidToolset.platform_list
+    $platformList | ForEach-Object {
+        $platformTestCases += @{ platformVersion = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    $buildToolsTestCases = @()
+    $buildToolsList = $androidToolset.build_tools
+    $buildToolsList | ForEach-Object {
+        $buildToolsTestCases += @{ buildToolsVersion = $_; installedPackages = $androidInstalledPackages }
+    }
+
+    It "Platform version <platformVersion> is installed" -TestCases $platformTestCases {
```

Comment:
I think we also need to validate `extra_list`, `addon_list`, `additional_tools`

Reply:
Fixed